x-pack/plugins/dashboard_enhanced/public/services/drilldowns/embeddable_to_dashboard_drilldown/embeddable_to_dashboard_drilldown.tsx

```diff
@@ -14,8 +14,8 @@ import {
   isTimeRange,
   Query,
   TimeRange,
-  extractTimeRange,
 } from '@kbn/data-plugin/public';
+import { extractTimeRange } from '@kbn/unified-search-plugin/public';
```

Comment:
not sure that `extractTimeRange` should be part of  static contract of `unified-search-plugin`. I vote for duplicating

Reply:
`extractTimeRange` uses `extractTimeFilter` internally, which I moved to `unified search` to get rid of deprecated methods. If I make a copy in the `extractTimeRange` plugin `Date`, then `extractTimeFilter` will also need to be returned there. and add it back as a deprecated method. Is it right?

Reply:
@ppisljar  What do you think about moving `extractTimeRange` and extractTimeFilter into `kbn-es-query` package? Or we should refactor that somehow? I just see some plugins still use that methods and it's definitely not right to move it into `unified-search-plugin`

Reply:
cc @lukasolson i agree unified search plugin is not the right place for this and that moving to a package probably makes sense